src/useq/_plate.py

```diff
@@ -368,18 +366,24 @@ def affine_transform(self) -> np.ndarray:
         """Return transformation matrix.
 
         This includes:
-        1. scaling by plate.well_spacing
+        1. translation to a1_center_xy
         2. rotation by rotation_matrix
-        3. translation to a1_center_xy
+        3. scaling by plate.well_spacing
```

Comment:
actually these should stay in the order they were.  when you compose transforms (like `translation @ rotation @ scaling`) it goes from right to left

Reply:
yup sorry